conference.js

```diff
@@ -483,7 +487,19 @@ export default {
         let tryCreateLocalTracks;
 
         // FIXME the logic about trying to go audio only on error is duplicated
-        if (options.startScreenSharing) {
+        if (options.startAudioOnly) {
+            tryCreateLocalTracks
+                = createLocalTracks({ devices: ['audio'] }, true)
+                    .catch(err => {
+                        audioOnlyError = err;
+
+                        return [];
+                    });
+            // Enable audio only mode
+            if (!this.isAudioOnly()) {
```

Comment:
you mean the whole if or just the check ?
I just find it more correct given that it's a toggle function and not "set". Do you want me to remove it ?